lms/djangoapps/certificates/management/commands/fix_ungraded_certs.py

```diff
@@ -23,32 +24,40 @@ class Command(BaseCommand):
     and grade them.
     """
 
-    option_list = BaseCommand.option_list + (
-        make_option(
+    def add_arguments(self, parser):
+        parser.add_argument(
             '-n',
             '--noop',
             action='store_true',
             dest='noop',
             default=False,
             help="Print but do not update the GeneratedCertificate table"
-        ),
-        make_option(
+        )
+
+        parser.add_argument(
             '-c',
             '--course',
             metavar='COURSE_ID',
             dest='course',
-            default=False,
+            required=True,
             help='Grade ungraded users for this course'
-        ),
-    )
+        )
 
     def handle(self, *args, **options):
         course_id = options['course']
         log.info('Fetching ungraded students for %s.', course_id)
+
+        # I got an error in testing that course wasn't an OpaqueKey. Not sure if this is an old command
+        # or if we're calling it programmatically with OpaqueKey types, but I attempted to handle it here.
+        if isinstance(options['course'], types.StringType):
```

Comment:
The documentation for the `types` module indicates that this usage is deprecated, and Python 3 removes `types.StringType`.  Maybe use `six.string_types` instead?  There's another usage of this below.